src/app/screens/home/balanceCard/index.tsx

```diff
@@ -97,27 +97,26 @@ function BalanceCard(props: BalanceCardProps) {
           <CurrencyText>{fiatCurrency}</CurrencyText>
         </CurrencyCard>
       </RowContainer>
-      <BalanceContainer>
-        {isLoading ? (
-          <BarLoaderContainer>
-            <BarLoader loaderSize={LoaderSize.LARGE} />
-          </BarLoaderContainer>
-        ) : (
+      {isLoading ? (
+        <BarLoaderContainer>
+          <BarLoader loaderSize={LoaderSize.LARGE} />
+        </BarLoaderContainer>
+      ) : (
+        <BalanceContainer>
           <NumericFormat
             value={calculateTotalBalance()}
             displayType="text"
             prefix={`${currencySymbolMap[fiatCurrency]}`}
             thousandSeparator
             renderText={(value: string) => <BalanceAmountText>{value}</BalanceAmountText>}
           />
-        )}
-
-        {isRefetching && !isLoading && (
-          <ReloadContainer>
-            <MoonLoader color="white" size={16} />
-          </ReloadContainer>
-        )}
-      </BalanceContainer>
+          {isRefetching && (
+            <ReloadContainer>
+              <MoonLoader color="white" size={16} />
+            </ReloadContainer>
+          )}
+        </BalanceContainer>
```

Comment:
The scope of this balance container was causing the main issue of the loader not showing.

Reply:
interesting